app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%=
+    per_page_options = [5, 10, 15, 25, 50]
+    per_page_options << Setting[:entries_per_page].to_i
+    per_page_options << params[:per_page].to_i unless params[:per_page].blank?
```

Comment:
updated https://github.com/lizagilman/foreman/commit/dd452b9aafcb1bb76d3265ff56a602a89fffc05b, if there isn't anything missing there, should I add it to this pr?
@ohadlevy